example/res/values/strings.xml

```diff
@@ -125,4 +125,8 @@
     <string name="google_pay_merchant_country_code">Merchant Country Code</string>
     <string name="google_pay_merchant_country_code_default">US</string>
 
+    <!--  US Bank Account  -->
+    <string name="confirm_with_us_bank_account_entry_point">US Bank Account Example</string>
```

Comment:
I think we should be a bit clearer on the naming here. This [PR](https://github.com/stripe/stripe-android/pull/4726) already adds an activity with the same name. We could name this one US Bank Account with Connections Example or something?

Reply:
https://github.com/stripe/stripe-android/pull/4732/commits/e05f38931910faf73a8a3a4a3d348a36e22cc559